QA/PoroDevTest/src/main/java/common/ui_setup/pom_setup/PoroDevPom/LoginPage.java

```diff
@@ -1,55 +1,76 @@
 package common.ui_setup.pom_setup.PoroDevPom;
 
+import common.ui_setup.DriverSetup;
 import common.ui_setup.SetupConstants;
 import common.ui_setup.pom_setup.BasePage;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
-import java.io.IOException;
+import org.testng.Assert;
+
+import java.util.List;
 
 public class LoginPage extends BasePage {
 
     public LoginPage(WebDriver driver) {
         super(driver);
-        driver.get(SetupConstants.BASE_URL);
+    }
+
+    public LoginPage(WebDriver driver, String url) {
+        super(driver);
+        DriverSetup.navigateToUrl(SetupConstants.BASE_URL, driver);
     }
 
     //element section
-    @FindBy(id = "basic_email")
-    WebElement emailField;
-    @FindBy(id = "basic_password")
-    WebElement passField;
+    @FindBy(name = "email")
+    WebElement we_emailInputField;
+    @FindBy(name = "password")
+    WebElement we_passInputField;
     @FindBy(className = "ant-btn-primary")
-    WebElement loginButton;
+    WebElement we_loginButton;
     //ASSERTION ELEMENTS
-    @FindBy(xpath = "//div[contains(text(),'Please input your username')]")
-    public WebElement emailErrorMsg;
-    @FindBy(xpath = "//div[contains(text(),'Please input your password')]")
-    public WebElement passErrorMsg;
-    @FindBy(xpath = "//span[contains(text(),'Request failed with status code 401')]")
-    public WebElement errorAfterBadCredentials;
-    @FindBy(className = "hvVmBA")
-    public WebElement userFirstNameAfterLogin;
+    @FindBy(xpath = "//body/div[@id='root']/div[1]/div[1]/form[1]/div[1]/span[2]")
```

Comment:
Absolute xpath locators should be avoided. U can consult FE to add test attribute in order to achieve easier maintenance and increase readability